rollbar-java/src/integTest/java/com/rollbar/notifier/RollbarITest.java

```diff
@@ -182,11 +179,11 @@ public void shouldSendSuccessfullyMessagePayloadThroughProxy() {
 
     wireMockProxy.register(
         post(urlMatching(".*"))
-        .willReturn(aResponse().proxiedFrom("http://localhost:8089/")));
+        .willReturn(aResponse().proxiedFrom("http://localhost:" + PORT + "/")));
 
     // Note that this url doesn't end up with the slash char, it seems is getting removed after
```

Comment:
I guess this comment is not valid anymore?

Reply:
Well spotted! The commend is gone now. 
It seems the dropped trailing slash was a bug in Wiremock's proxy that they fixed.